draft-wqb-tvr-applicability.md

```diff
@@ -523,6 +523,140 @@ network nodes can be selectively disabled or enabled based on traffic patterns,
+
+### Architecture Example in Distributed Generation
+
+In the distributed schedule generation，the Schedule Service Requester in {{arch}}
+can be network administrator, and the Scheduled Service Responders are the network
+devices, the relationship of network administrator and network devices are shown in
+{{arch-example}}. In this mode, the generation and execution of schedules are both
+on the same devices，so it does not involve the schedule distribution process.
+
+~~~~
+          +-----------------------------------------------------+
+          |              Network Administrator                  |
+          +-+--------------^------------------+---------------^-+
+            |              |                  |               |
+            |Request       |Response          |Request        |Response
+            |              |                  |               |
+    +-------v--------------+-------+    +-----v---------------+--------+
+    |       Network Device A       |    |       Network Device B       |
+    |                              |    |                              |
+    |   +---------+  +---------+   |    |   +---------+  +---------+   |
+    |   | Schedule|  | Conflict|   |    |   |         |  |         |   |
+    |   | Manager |  | Resolver|   |    |   | Manager |  | Resolver|   |
+    |   +---------+  +---------+   |    |   +---------+  +---------+   |  …
+    |                              |    |                              |
+    |   +---------+  +---------+   |    |   +---------+  +---------+   |
+    |   | Resource|  | Policy  |   |    |   | Resource|  | Policy  |   |
+    |   | Manager |  | Engine  |   |    |   | Manager |  | Engine  |   |
+    |   +---------+  +---------+   |    |   +---------+  +---------+   |
+    |                              |    |                              |
+    +------------------------------+    +------------------------------+
+~~~~
+{: #arch-example title="An Architecture Example for Distributed Schedule Generation Scenario" artwork-align="center"}
+
+### Example Procedures
+
+
+#### Building traffic profiles
+
+The first step to perform schedules in a tidal network is to analysis the traffic patterns at different
+network devices and links comprehensively and then establishing clear tidal points for lower and upper
+network traffic. It should be noted that the change regularity of traffic may different at different time
+(for example the traffic regularity in workday and weekend may totally different), the selection of tidal
+points should take full count of these factors. How to analyze the traffic patterns and determine the
+tidal points are outside the scope of this document.
+
+#### Estabilish Minimum and Peak Topology
+
+An algorithm is required to calculate the minimum and peak topology to service expected demand at different
+time slot. The calculation algorithm for the topology is outside the scope of this document.
+
+#### Generating Schedule
```

Comment:
What is the relation between this section and above sections “Distributed Generation” and  “centralized Generation”？

Reply:
Both "Distributed Generation" and "Centralized Generation" need the same Schedule Generation Prodecures, the only difference between "Distributed Generation" and "Centralized Generation"  is that "Centralized Generation" may need to distribute schedules to the execution devices.